mypy/plugins/attrs.py

```diff
@@ -367,6 +398,11 @@ def _attribute_from_attrib_maker(ctx: 'mypy.plugin.ClassDefContext',
 
     # Read all the arguments from the call.
     init = _get_bool_argument(ctx, rvalue, 'init', True)
+    kw_only |= _get_bool_argument(ctx, rvalue, 'kw_only', False)
```

Comment:
I noticed this (a bit counterintuitive) behavior:
```python
import attr
@attr.s(kw_only=True)
class A:
    a = attr.ib(kw_only=False)

A(1)  # Error
```
The logic here captures it correctly, but I would add a short comment that this is intentional.

Reply:
I filed an issue in attrs to see if this is expected: https://github.com/python-attrs/attrs/issues/481